rfcs/text/0012-orchestrator-field-set.md

```diff
@@ -19,9 +19,74 @@ add a new schema rather than change existing material.
 
 ## Fields
 
-<!--
-Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
--->
+The proposed change adds six fields, as described below:
+
+```
+---
+- name: orchestrator
+  title: Orchestrator
+  group: 2
+  short: Fields relevant to container orchestrators.
+  description: >
+    Fields that describe the resources which container orchestrators manage or
+    act upon.
+  type: group
+  fields:
+    - name: cluster
+      level: extended
+      type: object
+      description: >
+        Orchestrator cluster details.
+
+    - name: type
+      level: extended
+      type: keyword
+      example: Nomad, Kubernetes
+      description: >
+        Orchestrator cluster type.
```

Comment:
Use lowercased keywords. The example should include a real valid example, if you want to mention more possible cases use the description, something like this:
```suggestion
      example: kubernetes
      description: >
        Orchestrator cluster type (e.g. kubernetes, nomad or cloudfoundry).
```